Core/GDCore/Extensions/Builtin/AudioExtension.cpp

```diff
@@ -146,69 +146,83 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAudioExtension(
       .AddAction("ModVolumeSoundCanal",
                  _("Volume of the sound on a channel"),
                  _("This action modifies the volume of the sound on the "
-                   "specified channel. The volume is between 0 and 100."),
+                   "specified channel."),
                  _("the volume of the sound on channel _PARAM1_"),
                  _("Sounds on channels"),
                  "res/actions/sonVolume24.png",
                  "res/actions/sonVolume.png")
       .AddCodeOnlyParameter("currentScene", "")
       .AddParameter("expression", _("Channel identifier"))
-      .UseStandardOperatorParameters("number")
+      .UseStandardOperatorParameters(
+          "number",
+          ParameterOptions::MakeNewOptions().SetDescription(
+              _("Volume (0-100)")))
```

Comment:
I wonder if the method could be directly in the metadata to make the declaration faster to write and easier to read.
What do you think about renaming `UseStandardOperatorParameters` to something like `AddOperatorAndValueParameters` and declaring methods like `SetLastParameterDescription`?

Reply:
I was thinking initially of using SetLastParameterDescription, but I have the feeling that it lacks some robustness... what if, one day for some reason, the value is not the "last parameter" anymore?
What if the parameter options include something like an acceptable range, that would have an impact on what operator is valid to use?